applications/GeoMechanicsApplication/custom_processes/README.md

```diff
@@ -59,5 +59,72 @@ Where the `model_part_name` should contain the name of the model part where the
 
 When this process is added to the `ProjectParameters.json`, the variables specified in `list_of_variables` can be exported as nodal output (e.g. as `nodal_results` in the `GiDOutputProcess`). 
 
+## $K_0$ procedure process
+For the initialization of an in-situ stress field, the $K_0$ procedure derives the horizontal effective stresses from a field of vertical effective stresses.
+Pre-requisite is a computed stress field with the desired normal effective stresses in the direction indicated with "K0_MAIN_DIRECTION". The normal effective stress in "K0_MAIN_DIRECTION" remains as is. Effective normal stresses in the other two directions are affected by the $K_0$ value, all shear stresses are erased.
+
+
+Depending on the given input parameters, the following scheme is adapted for computation of the $K_0$ value.
+$K_{0_{NC}}$ is gotten from either "K0_NC" the material input file or by computation from input of "INDEX_OF_UMAT_PHI_PARAMETER" and "UMAT_PARAMETERS"
+
+$$K_{0_{NC}} = 1.0 - \sin \phi$$
+
+When "OCR" and optionally "POISSON_UNLOADING_RELOADING" are supplied, the normal consolidation value $K_{0_{NC}}$ is modified:
+
+$$K_0 = OCR \cdot K_{0_{NC}} +  \frac{\nu_{ur}}{1 - \nu_{ur}} ( OCR - 1 )$$
+
+$$\sigma^{'}_{initial} = \begin{bmatrix} {K_0 \cdot \sigma^{'}_{zz}} & 0 & 0 \\
```

Comment:
Shall we also change this one to $K_0^{nc}$ instead of K0?

Reply:
On line 76 it really should be the K_0 value, the adaptation by OCR and nu_{ur} of line 74 needs to be used here.